app/console.go

```diff
@@ -2,19 +2,29 @@ package app
 
 import (
 	"strconv"
+	"sync"
 
 	"github.com/abiosoft/ishell"
 	"github.com/ubclaunchpad/cumulus/blockchain"
-	"github.com/ubclaunchpad/cumulus/conn"
 	"github.com/ubclaunchpad/cumulus/peer"
 )
 
-var shell *ishell.Shell
+var (
+	shell     *ishell.Shell
+	peerStore *peer.PeerStore
```

Comment:
Yea I'm not really sure what to do about that. I guess I could make my own Shell type that is composed of ishell.Shell and the Peerstore. The thing is, there wouldn't really be much point because that Shell type wouldn't be used anywhere else and would only ever be instantiated once throughout the lifetime of the program.